core/scripts/vrfv2/testnet/main.go

```diff
@@ -925,3 +949,19 @@ func getRlpHeaders(ec *ethclient.Client, blockNumbers []*big.Int) (headers [][]b
 	}
 	return
 }
+
+// estimates gas limit before creating the transaction
```

Comment:
nit: comments on functions should start with the function name an end with period, eg:

// estimateGasLimit estimates the gas limit before ....